hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenCreateHandler.java

```diff
@@ -319,6 +318,15 @@ private void addCustomFeeCollectorKeys(
         for (final var customFee : customFeesList) {
             final var collector = customFee.feeCollectorAccountIdOrElse(AccountID.DEFAULT);
 
+            // Verify that the collector either has a non-empty alias OR a mutable key
+            final var acctStore = context.createStore(ReadableAccountStore.class);
+            final var collectorAcct = acctStore.getAccountById(collector);
+            if (collectorAcct != null
```

Comment:
If this is necessary, it suggests we need to circle back to how we are adding hollow account signing requirements. 🤔 But probably better to do that in a dedicated issue.

Reply:
Sure, I'll open a new issue. For this I followed the mono code as best I could, but there could well be a better way to work the signing requirements.

Reply:
Agree with @tinker-michaelj . There were some mono-service enhancements happening when implementing this. It is possible we missed some signing requirements for hollow accounts.

Reply:
Opened #9195 to address this